templates/core/about/builds.html

```
@@ -7,72 +7,76 @@
+	</p>
+
+	<h4>Testing documentation builds locally</h4>
+	{%- set build_subcommand = docsrs_repo ~ "/blob/master/README.md#build-subcommand" -%}
+	<p>
+		The Doc.rs <a href="{{ build_subcommand | safe }}">README</a> describes how to build
+		unpublished crate documentation locally using the same build environment as the Doc.rs build agent.
 	</p>
 
-	<h4>Global sandbox limits</h4>
+	<h3>Diagnosing failed builds</h3>
 
+	<h4>Missing dependencies</h4>
 	<p>
-		All the builds on docs.rs are executed inside a sandbox with limited
-		resources. The current limits are the following:
+		Missing dependencies are a common reason for failed builds.
+		Docs.rs dependencies are managed through
+		<a href="https://github.com/rust-lang/crates-build-env">crates-build-env</a>.
+		See <a href="https://forge.rust-lang.org/docs-rs/add-dependencies.html">Forge</a> for instructions
+		on how to make a PR. You can always <a href="{{ docsrs_repo | safe }}/issues">file an issue</a>
+		when having trouble.
 	</p>
 
-	{{ macros::crate_limits(limits=limits) }}
+	<h4>Writing files to read-only directories</h4>
+	<p>
+		Most of the sandbox is a read-only file system, including the source directory of your crate and its dependencies.
+		If you <code>build.rs</code> generates files that are relevant for documentation, consider writing to the output directory which is passed as the environment variable <code>OUT_DIR</code>.
+	</p>
 
+	<h4>Hitting sandbox limits</h4>
 	<p>
-		If a build fails because it hit one of those limits please
+		All the builds on Docs.rs are executed inside a sandbox with limited
+		resources. If a build fails because it hit limits please
 		<a href="{{ docsrs_repo | safe }}/issues/new/choose">open an issue</a>
 		to get them increased for your crate.
-		Note that network access will not be enabled for any crate.
+		Network access will not be enabled for any crate.
+		The current limits are:
 	</p>
 
```

Could you add a "Requesting Rebuilds" section that outlines how to request a rebuild and why you can/can't?
You can request a rebuild by opening an issue on our repo, but we can rebuild crates that fail due to something that is *our* fault, like a missing dependency or docs.rs bug. Crates that hit resource limits *might* be rebuilt if we have the resources to do so (since our builder itself has finite resources), but failures due to networking will not be rebuilt

---

And if it needs a change to the source code a rebuild won't help (e.g. a dependency was fixed, but in a semver breaking bump).

---

Good idea. Will do so.